src/paradigma/tremor/feature_extraction.py

```diff
@@ -0,0 +1,431 @@
+import numpy as np
+import pandas as pd
+from scipy import signal
+from paradigma.constants import DataColumns
+
+def compute_welch_periodogram(
+        values: list,
+        window_type: str = 'hann',
+        sampling_frequency: int = 100,
+        segment_length_s: float = 3,
+        overlap: float = 0.8,
```

Comment:
Wellicht voor consistentie deze hernoemen naar bijv `overlap_s` (ook in de config file).

Reply:
Aangepast naar overlap_fraction